JavascriptMornings/Maitee
# Feature/gas station

## Review checklist

- [x] Add corresponding label

## Testing

Steps to test the change:
1. Run index.js
2. Check if the "Total" object works when getting the gas prices
3. Check if Maitee currentGallons change when buying gas 
4. Check if when buying gas if does not go over of the car MaxGallons
5. Be Happy 

## Additional notes
N/A
## Deploy checklist
